LayoutTests/media/track/track-forced-subtitles-in-band.html

```diff
@@ -69,29 +70,44 @@
                 testExpected("video.textTracks[8].mode", "disabled");
                 consoleWrite("");
 
-                waitForEventsAndCall([[video, 'seeked'], [video.textTracks[3], 'cuechange']], testVisibleCue);
+                testExpected("video.textTracks[9].language", "es");
+                testExpected("video.textTracks[9].kind", "chapters");
+                testExpected("video.textTracks[9].mode", "hidden");
+                consoleWrite("");
+
+                testExpected("video.textTracks[10].language", "en");
+                testExpected("video.textTracks[10].kind", "chapters");
+                testExpected("video.textTracks[10].mode", "hidden");
+                consoleWrite("");
 
+                consoleWrite("<i>** A cue should become active when current time is within its active range<" + "/i>");
                 run("video.currentTime = 9.5");
-            }
+                let result = await Promise.race([
+                    Promise.all([
+                        waitFor(video, 'seeked', true), 
+                        waitFor(video.textTracks[3], 'cuechange', true)
+                    ]),
+                    new Promise((resolve, reject) => { setTimeout(resolve, 5000, 'timeout'); })
+                ]);
 
-            function setup()
-            {
-                findMediaElement();
-                run("internals.settings.setShouldDisplayTrackKind('Captions', false)");
-                run("internals.settings.setShouldDisplayTrackKind('Subtitles', true)");
-                run("internals.setUserPreferredLanguages(['en'])");
-                run("internals.setPrimaryAudioTrackLanguageOverride('fr')");
-                run("internals.setCaptionDisplayMode('ForcedOnly')");
-                video.src = '../content/CC+Subtitles.mov';
+                if (result === 'timeout') {
+                    consoleWrite('');
+                    failTest('Test timed out.');
+                }
 
-                waitForEvent('canplaythrough', testTracks);
-            }
+                consoleWrite('EVENT(seeked)');
+                consoleWrite('EVENT(cuechange)');
+
+                testExpected('video.textTracks[3].activeCues.length === 1', true);
```

Comment:
Nit: curious why you changed this to a "=== true" test. Is there a hole in our `testExpected` implementation that makes this necessary?

Reply:
The previous test was `testExpected("video.textTracks[3].activeCues.length", 1)`. I added the `.length === 1` because only one of the the text track's cues should be active at 9.5s.